media/css/firefox/windows-64-bit.scss

```diff
@@ -0,0 +1,76 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/components/article';
+@import '../../protocol/css/components/call-out';
+
+h1, h2 {
+    @include open-sans;
+    @include text-display-lg;
+}
+
+.mzp-c-hero {
+    background: #33124e;
+    color: #fff;
+    display: grid;
```

Comment:
This could do with some fallback styling for browsers that don't support grid?

<img width="1660" alt="screenshot 2019-02-21 at 14 14 48" src="https://user-images.githubusercontent.com/400117/53175049-1bb1db80-35e3-11e9-92d1-50f6b2fdaa4b.png">